Apps/WebClient/src/ClientApp/src/components/navmenu/sidebar.vue

```diff
@@ -540,6 +618,22 @@ export default class SidebarComponent extends Vue {
         EventBus.$emit("timelineCreateNote");
     }
 
+    private dismissNoteNotification() {
+        console.log("Dismissing Note Notification...");
+        this.userProfileService
+            .createUserPreference({
```

Comment:
Looking deeper this looks like it will attempt to insert a new user preferences. Instead I believe we need to update them. Assuming from the front end that the preferences already existed.

Reply:
for this requirement, once user dismisses this note popover, this notification will never display again. Next time when user comes back, this flag will be true as we grab its value for db right?

Reply:
Right, but thats leaking implementation details to the front end. The webclient does not need to know how the state of the preference is set since it only needs to read it and update it. Having it being created or updated is a choice that the service or delegate will do.

Reply:
correct, normally this should be implemented on the Db Delegate level right? Or should we do the check at the Server Service level as well?

Reply:
btw, I recall in other stories, we do the check on DBDelegate, if it is true, we can confirm the check as a part of 8626: create user preference delegate ?